amity.py

```diff
@@ -0,0 +1,339 @@
+
+from app.staff import Staff
+from app.fellow import Fellow
+from app.office import Office
+from app.person import Person
+from app.livingspace import LivingSpace
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import (create_engine, Sequence, Column, Integer,
+                        String, ForeignKey)
+from sqlalchemy.ext.declarative import declarative_base
+
+
+Base = declarative_base()
+
+
+class User(Base):
+    '''class mapper for table user'''
+    __tablename__ = 'user'
+    id = Column(Integer, Sequence('user_id_seq'), primary_key=True)
+    person_name = Column(String(30))
+    type_person = Column(String(30))
+
+    def __repr__(self):
+        return "<User(person_name='%s', type_person='%s')>" % (
+            self.type_person, self.type_person)
+
+
+class Room(Base):
+    '''class mapper for table room'''
+    __tablename__ = 'room'
+    id = Column(Integer, Sequence('room_id_seq'), primary_key=True)
+    room_type = Column(String(20))
+    room_name = Column(String(30))
+    occupant1 = Column(String(30), ForeignKey('user.id'))
+    occupant2 = Column(String(30), ForeignKey('user.id'))
+    occupant3 = Column(String(30), ForeignKey('user.id'))
+    occupant4 = Column(String(30), ForeignKey('user.id'))
+    occupant5 = Column(String(30), ForeignKey('user.id'))
+    occupant6 = Column(String(30), ForeignKey('user.id'))
+
+    def __repr__(self):
+        if self.room_type.lower() == 'office':
+            return """<Room(room name='%s', occupant1='%s',
+            occupant2='%s', occupant3='%s', occupant4='%s',
+            occupant5='%s', occupant6='%s')>""" % (self.room_name,
+                                                   self.occupant1,
+                                                   self.occupant2,
+                                                   self.occupant3,
+                                                   self.occupant4,
+                                                   self.occupant5,
+                                                   self.occupant6)
+
+        return """<Room(room name='%s',
+         occupant1='%s',occupant2='%s',
+          occupant3='%s', occupant4='%s')>""" % (self.room_name,
+                                                 self.occupant1,
+                                                 self.occupant2,
+                                                 self.occupant3,
+                                                 self.occupant4)
+
+
+class Amity(object):
+
+    def __init__(self):
+        self.engine = ''
+        self.felo = Fellow()
+        self.staf = Staff()
+        self.livin = LivingSpace()
+        self.offy = Office()
```

Comment:
have better variable names..
`self.fellow, self.staff, self.living_space and self.office`